content/blog/2016/2016-08-25-sauce-pipeline.adoc

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: "Running Sauce On-Demand in Pipeline"
+tags:
+- pipeline
+- plugins
+- saucelabs
+- selenium
+- nightwatch
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at Cloudbees.
+
+The link:https://wiki.jenkins-ci.org/display/JENKINS/Sauce+OnDemand+Plugin[Sauce OnDemand Plugin]
+let's your Jenkins pipelines use Sauce Labs to run your Selenium tests on multiple browsers and platforms.
```

Comment:
s/let's/lets/

Also, is it "On-Demand" or "OnDemand", the post is somewhat inconsistent here.


This sentence feels like a little bit of proper noun soup.  Perhaps breaking it up into two separate statements: "Testing web applications across multiple browsers on different platforms can be challenging even for smaller applications. With Jenkins and the Sauce On-Demand plugin, you can wrangle that complexity by defining your Pipeline as Code"

(something like that?)